src/lib/create/from-string.js

```diff
@@ -192,10 +192,9 @@ export function configFromRFC2822(config) {
     }
 }
 
-// date from iso format or fallback
+// date from 1) ASP.NET, 2) ISO, 3) RFC 2822 formats, or 4) optional fallback if parsing isn't strict
 export function configFromString(config) {
     var matched = aspNetJsonRegex.exec(config._i);
```

Comment:
Why is ASP.NET used as the variable name?

Is this so that moment supports ASP.NET dates?
The variable is setup on line 45

var aspNetJsonRegex = /^\/?Date\((\-?\d+)/i;
it is basically a regex that matches a string like this:
/Date(9897979

Not a maintainer, just adding to the conversation.